analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/CookieShouldBeSecure_Net.cs

```diff
@@ -1,26 +0,0 @@
-﻿using System;
```

Comment:
History for this file will be probably lost (I'm not sure if squash merge will recognize the rename). I'm not sure if we can do something about it, but as general, it's good to rename files in a single commit to avoid git confusion when file is renamed and modified

Reply:
Ok, will put more attention into it.